resources/airflow.rb

```diff
@@ -67,6 +67,9 @@ def self.apply(model, runner, infil, mech_vent, nat_vent, duct_systems,
     pbar_sensor = OpenStudio::Model::EnergyManagementSystemSensor.new(model, "Site Outdoor Air Barometric Pressure")
     pbar_sensor.setName("out pb s")
 
+    wout_sensor = OpenStudio::Model::EnergyManagementSystemSensor.new(model, "Site Outdoor Air Humidity Ratio")
+    wout_sensor.setName("out wt s")
```

Comment:
Nice. You found that we already have a sensor for this and made it more global in scope to prevent duplicate sensors.